### DISCUSSION TITLE
Enable ignorecase in regex expression for Filter expert

### DISCUSSION THREAD

**Discussion by ludoComp9** [certtools/intelmq#2344] [Support] [answered]
Hello,

I would like to search specific keyword in `source.as_name` (such as `google`) but I would like to match keyword ignoring case format. Any solution to do it through Filter popup from IntelMQ WebUI or need to custom filter expert script ?

Regards

**Comment by sebix** [comment, marked as answer]
You can use the [filter expert](https://intelmq.readthedocs.io/en/latest/user/bots.html#filter)'s regex function and `[gG][oO][oO][gG][lL][eE]`.
Or the [sieve expert](https://intelmq.readthedocs.io/en/latest/user/bots.html#sieve).

No need to write your own bot :)
